docs/apis/ROI360/true_revenue_api.yaml

```diff
@@ -184,6 +185,320 @@ paths:
           type: string
         description: "App ID of the app for which you are getting tax rate rule catalog"
         example: id123456789
+      - in: query
+        name: format
+        required: false
+        schema:
+          type: string
+          enum: [csv, json]
+          default: json
+        description: "The output format of the tax catalog data. Defaults to 'json'."
+        example: json
+      - in: query
+        name: tax_name
+        required: false
+        schema:
+          type: string
+        description: "Filter catalog by the name appearing in the customer invoice that describes the specific type of tax, such as sales, VAT, GST."
+        example: Sales
+      - in: query
+        name: tax_rate
+        required: false
+        schema:
+          type: number
+          format: double
+          minimum: 0
+          maximum: 100
```

Comment:
Add this pattern to enforce the double format. In this case the decimal point is optional

pattern: '^\d+(\.\d{1,4})?$'

Reply:
Nice! I just pushed a new commit with the fix